.mci.yml

```diff
@@ -284,7 +284,12 @@ functions:
                   fi
                   export CMAKE=${cmake}
 
-                  python -m virtualenv venv
+                  if [ -f /opt/mongodbtoolchain/v3/bin/python3 ]; then
```

Comment:
I would suggest this instead:
```
if ! python -m virtualenv 2>/dev/null; then
    /opt/mongodbtoolchain/v3/bin/python3 -m venv venv
else
    python -m virtualenv venv
fi
```

On a system without the virtualenv module, the command will exit with non-zero status:

```
roberto@debian:~$ python -m virtualenv
/usr/bin/python: No module named virtualenv
roberto@miami:~$ echo $?
1
```

Reply:
`python -m virtualenv` fails with `too few arguments`. So:
```
if ! python -m virtualenv 2>/dev/null; then   
then> echo "FAILED"                                                                                                                                               
then> else   
else> echo "SUCCESS"                                                                                                                                              
else> fi                                     
SystemExit: 2
FAILED
``` 
fails. But: 
```
python -m virtualenv venv
... output ...
echo $?
0
```
succeeds locally. Maybe I'm doing this wrongly?

Reply:
Interesting.  It might be better to go with `if ! python -m virtualenv venv 2>/dev/null; then`.  Then if the system does have the virtualenv module it doesn't fail with too few arguments.  Then the `else` branch can also be eliminated.

Reply:
Yeah, that looks cleaner.